app/assets/javascripts/user_contacts.js

```diff
@@ -0,0 +1,19 @@
+$(function(){
+  // Hide table row with message in case it was successfully deleted from DB
+  $('.delete_contact').on("ajax:success", function(event){
+      var current_tr = $( this ).parents('tr')[0];
```

Comment:
remove this extra space from `$( this )`
in many places you have it without space, and in some with